src/flask/config.py

```diff
@@ -245,7 +246,7 @@ def from_file(
             app.config.from_file("config.json", load=json.load)
 
             import toml
-            app.config.from_file("config.toml", load=toml.load)
+            app.config.from_file("config.toml", load=toml.load, text=True)
```

Comment:
this parameter is a absolute lie - when true you take away the text and drop a buffer

this should be called `binary`

and it should use the open mode

Reply:
Don't worry, I was getting there in my review.

Reply:
I addressed this in my review.